source/users/custom-metadata.txt

```diff
@@ -15,6 +16,11 @@ Enable Custom User Data
 Overview
 --------
 
+Through user accounts, you can store and access metadata and custom data for each user.
```

Comment:
i think this should be rephrased to be cleaer that 'metadata' is referring to auth provider-specific metadata. 

i'd recommend a small paragraph or perhaps an intro sentence w 2 bullet points.